setup.py

```diff
@@ -54,7 +54,8 @@
             'six',
             'librosa',
             'pydrive',
-            'scipy'
+            'scipy',
+            'scikit-learn >= 0.18'
```

Comment:
whoops! didn't realize I was missing the scikit-learn dependency all together :/